test/integration/k8s/otel/k8s_otel_traces_test.go

```diff
@@ -57,7 +57,7 @@ func TestTracesDecoration(t *testing.T) {
 					require.NotEmpty(t, trace.Spans)
 					sd := jaeger.DiffAsRegexp([]jaeger.Tag{
 						{Key: "k8s.pod.name", Type: "string", Value: "^testserver-.*"},
-						{Key: "k8s.container.name", Type: "string", Value: "testserver"},
+						{Key: "k8s.container.name", Type: "string", Value: "beyla"},
```

Comment:
Hmm, why do we now see Beyla as container name? I think we want the container name to be related to the instance we are monitoring. Why did this change?

Reply:
(the test was not passing using `testserver` it was expecting `beyla`)

Reply:
nice catch, this time a integration test catch a bug!